src/output/file_name.rs

```diff
@@ -179,7 +180,7 @@ impl<'a, 'dir, C: Colours> FileName<'a, 'dir, C> {
     /// This method returns some `TextCellContents`, rather than a `TextCell`,
     /// because for the last cell in a table, it doesn’t need to have its
     /// width calculated.
-    pub fn paint(&self) -> TextCellContents {
+    pub fn paint(&self, quotes: &Quotes) -> TextCellContents {
```

Comment:
Being that `Quotes` is an enum with no value, it takes up 1 byte, so it's actually more efficient to pass it by value.

Reply:
clippy told it to me too but as I didnt check it i didnt saw it fixed tho